draft-ietf-mls-architecture.md

```diff
@@ -815,14 +816,22 @@ MLS requires the following policies to be defined, which restrict the set of
 acceptable behavior in a group. These policies must be consistent between
 deployments for them to interoperate:
 
+- A policy on which ciphersuites are acceptable.
+- A policy on any mandatory or forbidden MLS extensions.
```

Comment:
This isn't necessary since we negotiate ciphersuites and extensions, see line 740. Keep in mind these are policies that need to align for interop. We need some overlapped support for ciphersuites and extensions but will negotiate the rest, so not an interop issue

Reply:
The DS in some contexts will have some requirements regarding ciphersuites and extensions.
For example, an organization which requires FIPS compliance cannot permit the use of any Edwards curves-based ciphersuites. Another might require a security level which allow p384, p521 and Ed448, but not Ed25519 or p256.
Likewise, a DS could forbid the use of the AppAck mechanism and require the content negotiation mechanism.
These are reasonable policy choices for legitimate operational or security reasons.